CMakeLists.txt

```diff
@@ -41,7 +41,13 @@ torch_mlir_add_llvm_external_project(
   TORCH_MLIR_DIALECTS
   ${CMAKE_CURRENT_SOURCE_DIR}/externals/llvm-external-projects/torch-mlir-dialects)
 
+torch_mlir_add_llvm_external_project(
```

Comment:
Can we add a top level TORCH_MLIR_MHLO (name can be anything @silvasean any suggestions ?) CMake flag that can enable / disable the MHLO backend ? This can be the big hammer in case we have to disable it for any reason (broken on macOS etc) .

Reply:
Yeah, it's quite a good suggestion. But it seems our codes are integrated with torch-mlir a little bit tightly. For example, we registered the torch-to-mhlo conversion pass in [/include/torch-mlir/Conversion/Passes.td](https://github.com/llvm/torch-mlir/pull/1025/commits/a20226b6f824d5a1aca64aeec79bd0443eaf4fa8#diff-e378cdef89f6a7ada41d204c78c1580725d8b5bb0d59dfb37b34cf0861fb5361). Do you have any good idea to separate them？